Source/Core/DolphinQt2/Resources.cpp

```diff
@@ -40,7 +56,9 @@ void Resources::Init()
     m_ratings.append(QPixmap(rating.prepend(sys_dir)));
 
   m_misc.append(QPixmap(QStringLiteral("nobanner.png").prepend(sys_dir)));
-  m_misc.append(QPixmap(QStringLiteral("dolphin_logo.png").prepend(sys_dir)));
+  m_misc.append(QPixmap(QStringLiteral("dolphin_logo%1.png")
+                            .arg(Resources::GetPreferredIconScale())
+                            .prepend(sys_dir)));
   m_misc.append(QPixmap(QStringLiteral("Dolphin.png").prepend(sys_dir)));
```

Comment:
You should make a 2x and 4x image using the `dolphin-emu.svg` in Data/, and then load that here.